node/blockchain/tests/test_models/test_signed_change_request_message/test_genesis.py

```diff
@@ -26,9 +26,9 @@ def test_create_from_alpha_account_root_file(primary_validator_key_pair, account
     assert message.type == Type.GENESIS
     assert message.account_lock == AccountLock(primary_validator_key_pair.public)
     assert len(message.accounts) == len(account_root_file)
-    assert message.accounts.keys() == account_root_file.keys()
+    assert list(message.accounts.keys()) == [a.lower() for a in account_root_file.keys()]
```

Comment:
this unittest must remain unchanged, since upper hex digits must be allowed for AlphaAccount

Reply:
I left test as is.